docs/admin/settings.rst

```diff
@@ -76,7 +76,7 @@ Global Settings
   messages in the browser too, so this must be deactivated in production.
 
 ``request_timeout`` :
-  Global timeout of the requests made to others engines in seconds. A bigger
+  Global timeout of the requests made to others engines in seconds.  A bigger
```

Comment:
Why is this extra space needed?

Reply:
Sorry for nitpicking, this is the ["double sentence spacing"](https://en.wikipedia.org/wiki/Sentence_spacing_in_digital_media) markup.  Some text processors like emacs use this _markup_ when formatting text blocks.  Normally I think, having only one space _is not a bug_ / nothing we need to fix. Here I fixed it since I reviewd the whole article.

If you think this is nitpicking, I can stop adding such additional spaces in the future. / sorry

Reply:
Thanks for your answer. I think it isn't a problem, just I was curious about the rationale behind.